lib/item_repository.rb

```diff
@@ -0,0 +1,93 @@
+require_relative "./item"
+require_relative "./order"
+require_relative "./shared_methods"
+
+class ItemRepository
+
+  #-----------------
```

Comment:
Minor - these comments do not add much to the code or documentation, maybe they could be removed?